goalgorilla/open_social
# Issue #3382576 by nkoporec: Disable revisions for groups

## Problem
When doing a clean install of the latest Open Social (from branch dev-main), revisions are enabled for groups and show up as a tab in the group itself.

## Solution
Disable revisions as this was not enabled before.

## Issue tracker
https://www.drupal.org/project/social/issues/3382576

## Theme issue tracker
*[Required if applicable] Paste a link to the drupal.org theme issue queue item, either from [socialbase](https://www.drupal.org/project/socialbase) or [socialblue](https://www.drupal.org/project/socialblue). If any other issue trackers were used, include links to those too.*

## How to test
1. Install clean OS
2. Go to admin/group/types/manage/closed_group
3. See that Create a new revision when a group is modified checkbox is **disabled**.


## Definition of done
### Before merge

### After merge

## Screenshots

Before:
![image](https://github.com/goalgorilla/open_social/assets/35064680/ea684acf-7c4e-4ef5-b938-cfc85ce694f6)

After:
![image](https://github.com/goalgorilla/open_social/assets/35064680/0c024e4d-190e-4055-ac2f-3745b9ec0a5e)


## Release notes
Disabled revisions for groups.

## Change Record


## Translations
